fpna/DataModel/DataStructure.ipynb

```diff
@@ -151,7 +208,12 @@
         {
             "cell_type": "code",
             "source": [
-                "public record BasicCashflow : SimpleCashflow {}"
+                "public record BasicCashflow : Cashflow, IBased",
+                "\n{",
+                "\n    public double InitialValue { get; init; }",
```

Comment:
InitialValue, Length, Shift and Shape belongs to the same type as they are all related on computation. 
I would move Shape from Cashflow to IBased.
@sbatka what do you think?

Reply:
agreed